
string board_name
geometry_msgs/PoseWithCovariance pose

uint8 num_detections
uint8 num_inliers

